php/utils.php

```diff
@@ -234,7 +234,7 @@ function locate_wp_config() {
 
 		if ( file_exists( ABSPATH . 'wp-config.php' ) ) {
 			$path = ABSPATH . 'wp-config.php';
-		} elseif ( file_exists( ABSPATH . '../wp-config.php' ) && ! file_exists( ABSPATH . '/../wp-settings.php' ) ) {
+		} elseif ( file_exists( dirname( ABSPATH ) . '../wp-config.php' ) && ! file_exists( dirname( ABSPATH ) . '/../wp-settings.php' ) ) {
```

Comment:
`dirname()` replaces the `..` subpath that traverses to the parent folder, so you need to remove this bit from `wp-config.php` file.